File: Moduuli 7/1.py
def get_season(month):
    if month in (12, 1, 2):
        return "winter"
    elif month in (3, 4, 5):
        return "spring"
    elif month in (6, 7, 8):
        return "summer"
    elif month in (9, 10, 11):
        return "autumn"
    return None

s = input("Enter the number of a month (1-12): ")
try:
    month = int(s)
    print("You entered:", month)
    season = get_season(month)
    if season:
        print("The season is", season + ".")
    else:
        print("Please enter a number between 1 and 12.")
except ValueError:
    print("Invalid input! Please enter an integer between 1 and 12.")

File: JSMODUL2/1/1.js
let numbers = [];
for (let i = 1; i <= 5; i++) {
    let input = prompt(`Enter number ${i}:`);
    numbers.push(Number(input));
}
console.log("Numbers in reverse order:");
for (let i = numbers.length - 1; i >= 0; i--) {
    console.log(numbers[i]);
}